tests/acceptance/features/webUIUserKeysType/userKeys.feature

```diff
@@ -11,11 +11,15 @@ Feature: encrypt files using user specific keys
       | brand-new-user   |
     And encryption has been enabled
 
-  @skip @issue-33
-  Scenario: encrypt using user keys based encryption
+  @issue-33
+  Scenario: encrypt all files using user keys based encryption via the occ command
+    Given these users have been initialized:
```

Comment:
For this scenario, the user needs to be first initialized so that the skeleton file ``textfile0.txt`` even exists.